lms/templates/problem.html

```diff
@@ -36,7 +36,10 @@ <h3 class="hd hd-2 problem-header" tabindex="-1">
     % endif
     % if answer_available:
     <span class="problem-action-button-wrapper">
-        <button class="show problem-action-btn btn-default btn-small"><span class="icon fa fa-exclamation-triangle" aria-hidden="true"></span><span class="show-label">${_('Show Answer')}</span></button>
+        <button class="show problem-action-btn btn-default btn-small">
+            <span class="icon fa fa-exclamation-triangle" aria-hidden="true"></span><span class="show-label">${_('Show Answer')}</span>
+            <span class="show-answer-tooltip sr">${_('To get credit, only select after Submit.')}</span>
```

Comment:
@alisan617 My suggestion, if not too long: "To get credit, select Show Answer only after you select Submit."